src/dtx/dtx_common.c

```diff
@@ -1618,7 +1617,11 @@ start_dtx_reindex_ult(struct ds_cont_child *cont)
 
 	D_ASSERT(cont != NULL);
 
-	if (cont->sc_dtx_reindex || cont->sc_dtx_reindex_abort)
+	/* Someone is trying to stop former DTX reindex ULT, wait until its done. */
+	while (cont->sc_dtx_reindex_abort)
+		ABT_thread_yield();
```

Comment:
please use dss_sleep to help the scheduler, instead of using yield directly.

Reply:
probably use cond signal here?

Reply:
This is relative rare race and the race window is very small, it is unnecessary to add new condition member for that.

Reply:
> please use dss_sleep to help the scheduler, instead of using yield directly.

Hm, the race window is very small, on the other hand, related logic maybe triggered when open container, I do not want to block it for too long time. So I do not know how long sleep is suitable, maybe just use "zero" as the input parameter, that is equal to ABT_thread_yield() but without more unnecessary CPU cycles?